lib/Controller/Fault.php

```diff
@@ -209,21 +210,13 @@ public function collect(Request $request, Response $response)
         // Send via Apache X-Sendfile header?
         if ($this->getConfig()->getSetting('SENDFILE_MODE') == 'Apache') {
             header("X-Sendfile: $tempFileName");
-            $response->withStatus(200);
         }
         // Send via Nginx X-Accel-Redirect?
         if ($this->getConfig()->getSetting('SENDFILE_MODE') == 'Nginx') {
             header("X-Accel-Redirect: /download/temp/" . basename($tempFileName));
-            $response->withStatus(200);
         }
 
-        // Return the file with PHP
```

Comment:
If you delete this, then it won't return the file for a CMS with sendfile mode off?

I wonder if you might want something like:

```
use GuzzleHttp\Psr7\Stream;
$response = $response->withBody(new Stream(fopen($tempFileName, 'r')));
```

Reply:
We most likely should use the `withHeader()` also, so that we don't modify the output directly